js/login.js

```diff
@@ -5,7 +5,22 @@ document.getElementById("manuallyIpBtn").addEventListener("click", manuallyIpNew
 let interval;
 const refreshTime = 1500;
 const postJsonObj = {devicetype: "Hue-Browser-Controller"};
-checkLocalStorage();
+checkQueryArgs().then(checkLocalStorage);
+
+function checkQueryArgs() {
+    const params = new URLSearchParams(window.location.search);
+    const token = params.get("token")
+    const ip = params.get("ip")
+    if (token && ip) {
+        const acc = {token,ip};
+        return connectionGood(acc).then(() => {
+            localStorage.setItem('hueAcc', JSON.stringify(acc));
+            window.location.search = "";
```

Comment:
I don't understand why the `window.location.search = "";` is necessary and if it's there I can't login using the URL parameters. So I will suggest removing it if it doesn't serve any other purpose.

Reply:
It is not strictly necessary. It's for the esthetics of the URL, preventing accidental reconnecting after explicitly disconnecting, and to not show token and network information when sharing the screen.

Reply:
That's a fair point but my issue is that it does not work in my Chrome browser if line 18 is not commented out...

Reply:
Wondering if this is caused by a refresh it triggers. If so, this could be the solution: https://stackoverflow.com/a/28609299/1455912

Reply:
Let me know if it works now

Reply:
Looks to be working fine now. Thanks for your contribution!